.rubocop.yml

```diff
@@ -20,6 +20,8 @@ AllCops:
     - 'activestorage/test/dummy/**/*'
     - 'actiontext/test/dummy/**/*'
     - 'tools/rail_inspector/test/fixtures/*'
+    - guides/source/debugging_rails_applications.md
+    - guides/source/active_support_instrumentation.md
```

Comment:
I explicitly excluded these files, because `guides/source/debugging_rails_applications.md` have a few debugging examples, like https://github.com/rails/rails/blob/7c03e4f1b7553084f34318a9efc2793137de36f4/guides/source/debugging_rails_applications.md?plain=1#L367-L383 These are invalid ruby, but removing `rb` from the markdown will disable syntax highlighting. 

guides/source/active_support_instrumentation.md has many lines like https://github.com/rails/rails/blob/7c03e4f1b7553084f34318a9efc2793137de36f4/guides/source/active_support_instrumentation.md?plain=1#L134 which also makes these snippets as invalid ruby, but syntax highlights in markdown.